easybuild/easyblocks/g/gromacs.py

```diff
@@ -209,6 +210,51 @@ def configure_step(self):
                 # to avoid that GROMACS finds and uses a system-wide CUDA compiler
                 self.cfg.update('configopts', "-DGMX_GPU=OFF")
 
+        # CP2K detection
+        cp2k_root = get_software_root('CP2K')
+        if self.cfg['cp2k'] and not cp2k_root:
+            msg = "The CP2K module needs to be loaded to build GROMACS with CP2K support."
+            raise EasyBuildError(msg)
+        elif cp2k_root and self.cfg['cp2k'] is False:
+            self.log.info('CP2K was found, but compilation without CP2K has been requested.')
+            cp2k_root = None
+
+        # enable CP2K support if CP2K is listed as a dependency
+        # and CP2K support is either explicitly enabled (cp2k = True) or unspecified ('cp2k' not defined)
+        if cp2k_root and (self.cfg['cp2k'] or self.cfg['cp2k'] is None):
```

Comment:
I actually missed this for plumed, but because of you explicitly setting cp2k_root to None, for `self.cfg['cp2k'] is False` you could simplify this to `if cp2k_root:`
or you can delete line 220 above of course.

Reply:
done